pkgs/development/python-modules/levenshtein/default.nix

```
@@ -44,11 +42,15 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [ "Levenshtein" ];
 
-  meta = with lib; {
+  meta = {
     description = "Functions for fast computation of Levenshtein distance and string similarity";
     homepage = "https://github.com/maxbachmann/Levenshtein";
-    changelog = "https://github.com/maxbachmann/Levenshtein/blob/${src.rev}/HISTORY.md";
-    license = licenses.gpl2Plus;
-    maintainers = with maintainers; [ fab ];
+    changelog = "https://github.com/maxbachmann/Levenshtein/blob/v${version}/HISTORY.md";
+    license = lib.licenses.gpl2Plus;
+    maintainers = with lib.maintainers; [ fab ];
+    # Issue with clang >= 19
+    # error: implicit instantiation of undefined template 'std::char_traits<unsigned int>'
+    # https://github.com/rapidfuzz/Levenshtein/issues/71
+    broken = stdenv.cc.isClang;
```

```suggestion
    broken = stdenv.cc.libcxx != null # ??
```
technically this is an issue with `libcxx` as clang with libstdc++ should be fine but I think using `stdenv.cc.libcxx` might just be a hack. what you have probably is good enough.